lib/email/receiver.rb

```diff
@@ -239,6 +244,18 @@ def wrap_body_in_quote(user_email)
       @body = "[quote=\"#{user_email}\"]\n#{@body}\n[/quote]"
     end
 
+    def create_post_action(email_log, type)
+      PostAction.act(email_log.user, email_log.post, type)
```

Comment:
Alright, I've applied a guardian call to the create_post_action and added a few specs around it; should be good for another look @eviltrout